### PR TITLE
text_on_img_200

### PR DESCRIPTION
**Pull request by jpascoe32fb**
adding background img for text_on_img_200
addressing the issue https://github.com/KwickerHub/WebCraftifyAI/issues/650

### CODE REVIEW

File: modules/text_on_image/text_on_image_200_John [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```
Comment (krushnarout) [inline modules/text_on_image/text_on_image_200_John]: @jpascoe32fb You must change the file name to `text_on_image_200_John.html`

**Review by krushnarout — CHANGES_REQUESTED** [review]

File: modules/text_on_image/text_on_image_200_John (hunk 1 above)
Comment (jpascoe32fb) [reply modules/text_on_image/text_on_image_200_John]: Whoops! Should be good now!

**Comment by krushnarout** [comment]
@jpascoe32fb Looks good to me

**Review by krushnarout — APPROVED** [review]

**Comment by krushnarout** [comment]
@jpascoe32fb Thanks for contributing

**Pull request merged**